testplan/common/utils/testing.py

```diff
@@ -140,26 +166,6 @@ def check_iterable(
     """
     _orig_act = _orig_act or actual
     _orig_exp = _orig_exp or expected
-
-    def render_mismatch(act, exp, full_path):
-        return (
-            "{linesep}"
-            'Mismatch: "{full_path}",{linesep}'
-            "Expected value: {expected}{linesep}"
-            "Actual value: {actual}{linesep}"
-            "Expected Data:{linesep}{exp_data}{linesep}"
-            "Actual Data:{linesep}{act_data}"
-        ).format(
-            full_path=full_path,
-            linesep=os.linesep,
-            expected=pprint.pformat(exp, indent=2),
-            actual=pprint.pformat(act, indent=2),
-            exp_data=pprint.pformat(_orig_exp, indent=2),
-            act_data=pprint.pformat(_orig_act, indent=2),
-        )
-
-    msg = render_mismatch(act=actual, exp=expected, full_path=curr_path)
```

Comment:
This doesn't need to be invoked on every single recursion. Another easy optimization to move this to where it _might_ be used.